doc/lsst.analysis.tools/index.rst

```diff
@@ -29,53 +26,32 @@ For a tutorial on working with
    action-types
    plot-types
 
+.. _lsst.analysis.tools-help:
+
 Need Help?
 ==========
 
-If you get stuck with ``analysis_tools`` then feel free to reach out to the ``#rubinobs-analysis-tools``
-channel on slack and hopefully someone will help you!
-
-
-.. _lsst.analysis.tools-contributing:
-
-Contributing
-============
-
-``lsst.analysis.tools`` is developed at https://github.com/lsst/analysis_tools.
-You can find Jira issues for this module under the `analysis_tools <https://jira.lsstcorp.org/issues/?jql=project%20%3D%20DM%20AND%20component%20%3D%20analysis_tools>`_ component.
-
-.. If there are topics related to developing this module (rather than using it), link to this from a toctree placed here.
+If you have any questions regarding ``analysis_tools`` it is recommended that you post your question on `The Community Forum <https://community.lsst.org/>`_.
```

Comment:
I just wanted to check that this was agreed on. Is someone assigned to look out for questions on Community, or do we expect that someone outside DRP is going to forward questions to us?

Reply:
My understanding is that the official line is to always redirect questions from the community onto Community. Slack is convenient, but also seen as a transient resource (i.e., not good for long-term information storage). Happy to punt on this one however if you would like to bring it up for a broader discussion?

Reply:
Update from Yusra via Slack:
> We can only direct staff to Slack. Your commissioning in-kinds are considered staff, so if you’re talking about comms to them in particular, you can direct them to Slack.  But if this is going on the [pipelines.lsst.io](http://pipelines.lsst.io/), you need to only direct them to community. That’s our only official user support.

Reply:
Removed the Slack component, and leaving Community here in this section accordingly.